sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql

```diff
@@ -49,7 +57,11 @@ first_seen AS (
   FROM
     `moz-fx-data-shared-prod`.fenix.baseline_clients_first_seen
   WHERE
-    submission_date >= "2021-01-01"
+    {% if is_init() %}
+      submission_date >= DATE("2021-01-01")
+    {% else %}
+      submission_date = @submission_date
+    {% endif %}
```

Comment:
The non-init query uses `>= "2021-01-01"` as well 
```suggestion
    submission_date >= DATE("2021-01-01")
```